.gitignore

```diff
@@ -22,4 +22,4 @@ DerivedData
 
 Carthage.pkg
 CarthageApp.pkg
-Carthage.xcodeproj
+Carthage.xcodeproj/
```

Comment:
This was incorrectly specified in my previous PR, so the generated project was accidentally checked in. I've removed it below.